test/connect_helpers_spec.rb

```diff
@@ -42,8 +47,11 @@ def api_error(code: 400, headers: {}, body: "")
     it "reports an error with details on timeout" do
       details = _("Make sure that the registration server is reachable and\n" \
         "the connection is reliable.")
+
+      expect(subject).to receive(:wrap_text).with("Details: #{details}", 76)
```

Comment:
The value `76` depends on the `80` value above, it would be nice to have `let(:screen_width) { 80 }` and use it mocking and the test.

Reply:
:1: